tap/tracker.py

```diff
@@ -118,6 +125,7 @@ def generate_tap_reports(self):
             if self.outdir:
                 combined_file = os.path.join(self.outdir, combined_file)
             with open(combined_file, 'w') as out_file:
+                self._write_tappy_version(out_file)
```

Comment:
I think we can support outputting the version info for the streaming mode too. Can you add some code to the end of the `__init__` method to something like the following?
```python
if self.streaming:
    self._write_tap_version(self.stream)
```

Reply:
Do you think this counts as a header row? One of the existing tests tests that no header is written when header is set to False.  Wondering whether the TAP version should printed if it is set to False - yes because it's separate to the header line that is generated for each test?

Reply:
No, I don't think it counts as a header row. The version row is part of the TAP spec. The header row is a concept internal to tappy to show test case names. I think the TAP version should be printed regardless of the boolean state of the header field.